erpnext/hr/doctype/expense_claim/expense_claim.py

```diff
@@ -112,11 +127,23 @@ def get_gl_entries(self):
 
 		# expense entries
 		for data in self.expenses:
+			amount = max(0 ,data.sanctioned_amount - data.sanctioned_tax)
```

Comment:
Why this? sanctioned_tax cannot be greater than sanctioned_amount, already validated.